mapper/overview_test.go

```diff
@@ -200,10 +200,18 @@ func TestOverview(t *testing.T) {
 		So(overview.ServiceMessage, ShouldEqual, sm)
 
 		So(overview.HasSDC, ShouldBeFalse)
+
+		So(overview.IsCustom, ShouldBeFalse)
+	})
+
+	Convey("test filter flex overview maps for custom datasets correctly", t, func() {
+		isCustom := true
+		overview := m.CreateFilterFlexOverview(filterJob, filterDims, dimDescriptions, pops, sdc, false, isCustom)
+		So(overview.Metadata.Title, ShouldEqual, "Create a custom dataset")
 	})
 
 	Convey("test truncation maps as expected", t, func() {
-		overview := m.CreateFilterFlexOverview(filterJob, filterDims, dimDescriptions, pops, sdc, false)
+		overview := m.CreateFilterFlexOverview(filterJob, filterDims, dimDescriptions, pops, sdc, false, false)
```

Comment:
Moving the custom conditional to the mapper will save updating all of these tests